packages/internal/src/__tests__/ast.test.ts

```diff
@@ -8,8 +10,8 @@ import {
   fileToAst,
 } from '../ast'
 
-jest.mock('@redwoodjs/project-config', () => {
-  const path = require('path')
+vi.mock('@redwoodjs/project-config', async () => {
+  const path = await import('path')
```

Comment:
Can the await import of path here be deduped in favor of the static one at the top?

Reply:
But also I know it was required before and principle of least change 😅

Reply:
Yeah makes sense to me! I was slightly worried about the hoisting but it's totally fine. Updated in [caeecc6](https://github.com/redwoodjs/redwood/pull/10162/commits/caeecc6c5d760b40c74a1220d9b88819c337dcac)